sql-common/client.c

```diff
@@ -2700,75 +2696,30 @@ static int ssl_verify_server_cert_san(X509 *server_cert,
 /*
   Check the server's (subject) Common Name against the
   hostname we connected to
-||||||| merged common ancestors
```

Comment:
This does not look right. `ssl_verify_server_cert` contains our Percona-specific extensions for validating host names with SAN-enabled SSL certificates. So, basically, we do better than upstream on platforms where there is no OpenSSL with native SAN validation. So, these changes should be preserved. See https://jira.percona.com/browse/PS-1076 for more details.

However, Oracle did really bad in https://github.com/mysql/mysql-server/commit/253d846f5904983bb4a4a64baa9237955c060237 with both clang-formatting `sql-common/client.c` /  `sql/session_tracker.cc` and fixing real problem `Bug #29297652: STACK OVERFLOW IN READ_OK_EX() WITH SESSION_STATE_CHANGED` in the same commit.
So, It looks like the proper way to do the merge would be to create one more prerequisite commit in this PR (just before the merge commit) in which you would just clang-format those 2 files.

Reply:
These conflicts are badly detected because of clang reformatting. All conflicts are resolved taking `HEAD` what keeps original Percona code.